pkg/connector/ble/ble.go

```diff
@@ -146,61 +230,51 @@ func NewConnection(ctx context.Context, vin string) (*Connection, error) {
 	}
 }
 
-func tryToConnect(ctx context.Context, vin string) (*Connection, error) {
+func tryToConnect(ctx context.Context, vin string, target Advertisement) (*Connection, error, bool) {
```

Comment:
This function fails the recently merged linter checks because `error` isn't the final return value. Please fix and rebase off of the latest version.

Reply:
Fixed linters